### PR TITLE
Attempt 2 Make class variables immutable

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Attempt 2 at #187

### CODE REVIEW

File: asdf_astropy/converters/unit/tests/test_magunit.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def create_magunits():
             else:
                 magunits.append(magunit)
 
-    return magunits
+    return set(magunits)
```
Comment (perrygreenfield) [inline asdf_astropy/converters/unit/tests/test_magunit.py:37]: For immutability shouldn't `frozenset()` be used?

**Review by perrygreenfield — COMMENTED** [review]
Other than one comment on sets, LGTM. But if this should be the general approach now, we should change the converter examples too in readthedocs.

File: asdf_astropy/converters/unit/tests/test_magunit.py (hunk 1 above)
Comment (WilliamJamieson) [inline asdf_astropy/converters/unit/tests/test_magunit.py:37]:
```suggestion
    return frozenset(magunits)
```

**Review by perrygreenfield — APPROVED** [review]
LGTM

**Pull request merged**